src/helloworld.hpp

```diff
@@ -0,0 +1,12 @@
+#ifndef _HELLOWORLD_H_
+#define _HELLOWORLD_H_
+
+#include <SDKDDKVer.h>
+#include <windows.h>
```

Comment:
you shouldn't need `SDKDDKVer.h`, or `windows.h` in this file.
you'll only need them in `dllmain.cpp`